### DISCUSSION TITLE
Xamarin Forms UWP "Access to the path 'C:\\xxx' is denied."

### DISCUSSION THREAD

**Discussion by GUOLDEV** [reactiveui/ReactiveUI#3730] [General]
I am getting an exception error "Access to the path 'C:\\xxx' is denied."  when trying to get data using "await BlobCache.UserAccount.GetAllObjects<T>();"
Is there a permission config that I am missing?

**Comment by glennawatson** [comment]
Was this discussion point meant for akavache?

**Reply by GUOLDEV** [reply]
Yes, sorry! wrong place
